src/pages/not-found/index.jsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import { useTranslation } from "react-i18next";
+
+export default function Insights() {
+  const { t } = useTranslation();
+  return (
+    <center>
```

Comment:
I was not aware of the existence of this element. It seems to be [deprecated](https://developer.mozilla.org/fr/docs/Web/HTML/Element/center)

`<Box sx={{textAlign: 'center'}}` is the recommended way to get similar result